central/auth/service/service_impl_test.go

```diff
@@ -126,3 +182,464 @@ func (s *authServiceAccessControlTestSuite) TestAuthServiceResponse() {
 		})
 	}
 }
+
+func (s *authServiceAccessControlTestSuite) TestValidateAuthMachineToMachineConfig() {
+	testCases := []struct {
+		config      *v1.AuthMachineToMachineConfig
+		skipIDCheck bool
+		err         error
+	}{
+		{
+			config: nil,
+			err:    errox.InvalidArgs,
+		},
+		{
+			config: &v1.AuthMachineToMachineConfig{},
+			err:    errox.InvalidArgs,
+		},
+		{
+			config: &v1.AuthMachineToMachineConfig{
+				Id: "some-id",
+			},
+			err: errox.InvalidArgs,
+		},
+		{
+			config: &v1.AuthMachineToMachineConfig{
+				Id:                      "some-id",
+				TokenExpirationDuration: "1",
+			},
+			err: errInvalidToken,
+		},
+		{
+			config: &v1.AuthMachineToMachineConfig{
+				Id:                      "some-id",
+				TokenExpirationDuration: "1s",
+			},
+			err: errox.InvalidArgs,
+		},
+		{
+			config: &v1.AuthMachineToMachineConfig{
+				Id:                      "some-id",
+				TokenExpirationDuration: "24h1s",
+			},
+			err: errox.InvalidArgs,
+		},
```

Comment:
Sorry, these still don't make sense to me. Is it `InvalidArgs` because `TokenExpirationDuration` is wrong or because `Issuer` is missing?